ParkingZoneApp/Services/IParkingZoneService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ParkingZoneApp.Models;
+
+namespace ParkingZoneApp.Services
+{
+    public interface IParkingZoneService
```

Comment:
Create Base IService interface and implement it using Service.cs. After that add ParkingZoneService and inherit from it like Repository